bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/basic/ActionConditions.java

```diff
@@ -77,8 +77,7 @@ public class ActionConditions {
      * @return RGB value + brightness as first byte
      */
     private static @Nullable JsonElement hsvToBRGB(@Nullable Command command, @Nullable JsonElement value) {
-        if (command != null && command instanceof HSBType) {
-            HSBType hsb = (HSBType) command;
+        if (command != null && command instanceof HSBType hsb) {
```

Comment:
Is this a good way to assign the variable?
It certainly does not contribute to the readability of the code

Reply:
null check is not needed as instanceof covers that check
```suggestion
        if (command instanceof HSBType hsb) {
```